package/MDAnalysis/coordinates/GRO.py

```diff
@@ -28,39 +28,6 @@
 `GRO format`_ which includes a conversion routine for the box.
 
 
-Writing GRO files
```

Comment:
So some docs have been "promoted" to the "loading your stuff" level.  Eg user friendly stuff like this